podcasts/New Search/Views/Results/PodcastCarouselView.swift

```diff
@@ -126,6 +126,15 @@ struct PodcastsCarouselView: View {
         }
     }
 
+    private func fillerPodcast() -> Podcast {
+        let podcast = Podcast()
+        podcast.title = ""
+        podcast.author = ""
+        podcast.uuid = ""
```

Comment:
Since the uuid is used to identify the view, should we return a random one to prevent any possible collisions?

```suggestion
        podcast.uuid = UUID().uuidString
```